torch/_prims/executor.py

```diff
@@ -10,7 +10,7 @@
 from torch.utils._pytree import tree_map
```

Comment:
@mruberry Please review my latest change in this file

Reply:
Looks OK to me but I'd sanity check with @IvanYashchuk and/or @kevinstephano